stork-kubernetes-quickstart/architecture.yml

```diff
@@ -0,0 +1,156 @@
+kind: Role
```

Comment:
it feels a bit weird that this file is called `architecture.yml`. Maybe `kubernetes-setup.yml`? Or `client-service.yml`?